app/src/main/res/layout/instagrammer_hover_item_list.xml

```diff
@@ -0,0 +1,25 @@
+<?xml version="1.0" encoding="utf-8"?>
+<RelativeLayout xmlns:android="http://schemas.android.com/apk/res/android"
+    xmlns:app="http://schemas.android.com/apk/res-auto"
+    android:layout_width="match_parent"
+    android:layout_height="match_parent">
+
+    <de.hdodenhof.circleimageview.CircleImageView xmlns:app="http://schemas.android.com/apk/res-auto"
+        android:id="@+id/user_avatar"
+        android:layout_width="55dip"
+        android:layout_height="55dp"
+        android:layout_centerInParent="true"
+        android:scaleType="centerCrop"
+        app:civ_border_color="#FFFFFF"
```

Comment:
It would be really nice if you start migrating these color resources to colors.xml (when all colors are cool for a final version)